### ISSUE TITLE
Add shortcut and menu item to hide/unhide all Container tabs.

### ISSUE THREAD

**Issue by jnachtigall** [mozilla/testpilot-containers#338]
Spin-off of https://github.com/mozilla/testpilot-containers/issues/331#issuecomment-284218202

It would be nice to quickly hide *all* (or certain container groups) quickly with a shortcut. Use case: Someone enters the room and you want to hide all Personal (and/or Shopping) container tabs.

Currently I can only hide a container one by one, but neither all at once and also not (all at once) with a shortcut:

![auswahl_071](https://cloud.githubusercontent.com/assets/3307404/23586439/10af24d8-0195-11e7-8587-bf614077d59a.png)

**Comment by jonathanKingston** [comment]
Thanks for the suggestion!
How would a user reopen the same container (probably shift + shortcut to reopen last)?
Do you have any suggestions on a shortcut?

**Comment by jnachtigall** [comment]
> How would a user reopen the same container (probably shift + shortcut to reopen last)?
> Do you have any suggestions on a shortcut?

Rethinking it, I guess this ticket is two issues in one:

1) Add a way to globally hide/unhide all containers. I think this could be done be adding an "eye icon toggle" next to the sort icon with a tooltip "Hide (or show/unhide) all containers" at:
![container_eye](https://cloud.githubusercontent.com/assets/3307404/23602219/bf08bfe8-024f-11e7-9eef-37947e1ecf08.png)

Shortcuts suggestion is very difficult because one has to make sure it is not conflicting with anything else on Browser, popular Add-ons or OS. Maybe `Ctrl+K Ctrl+<something-else>` (`K` for container for easy memorization). That is, `Ctrl+K` followed immediately by `Ctrl+<something-else>`. VSCode has these compound shortcodes ([link](https://code.visualstudio.com/docs/customization/keybindings)) and I really like these.

2. Add a shortcode to all individual containers, maybe `Ctrl+K Ctrl+<number>`, e.g.:
* `Ctrl+K Ctrl+1` to toggle Personal containers visibility (Hide/Show this container)
* `Ctrl+K Ctrl+2` to toggle Work containers visibility (Hide/Show this container)
* `Ctrl+K Ctrl+3` to toggle Finance containers visibility (Hide/Show this container)

But again really think twice before adding a shortcut because usually this cannot be undone/changed later easily ;)

**Comment by rubcap** [comment]
Hi, in addition to hiding _all_ containers, people using containers for tab management may want to switch between containers (without having to switch between windows).

Use case: "Let's hide my work tabs and do some reading for the next hour!"

Suggestion for UI:
![suggestion_container_ui](https://user-images.githubusercontent.com/26146350/28986743-86c6ae3e-7968-11e7-90b0-0a245b66c054.png)

PS.  "Always open in Default" doesn't make much sense, does it? A checkbox to activate the already default behavior?

**Comment by almet** [comment]
This UI makes a lot of sense. I didn't understood the  hide/show feature before reading an article explaining it to me. I even thought the feature had been removed.

Hope this UI will get some traction, thanks!

**Comment by kris7453** [comment]
> 
> 
> Hi, in addition to hiding _all_ containers, people using containers for tab management may want to switch between containers (without having to switch between windows).
> 
> Use case: "Let's hide my work tabs and do some reading for the next hour!"
> 
> Suggestion for UI:
> ![suggestion_container_ui](https://user-images.githubusercontent.com/26146350/28986743-86c6ae3e-7968-11e7-90b0-0a245b66c054.png)
> 
> PS. "Always open in Default" doesn't make much sense, does it? A checkbox to activate the already default behavior?

Yeah, I think too that suggested UI will be a greater option to hide/unhide button.
It is so cumbersome, to opening containers one by one to hide all containers.

I suggest too to add to tab context menu options to hide container to which the tab belongs for better management.
